modules/standard/IO.chpl

```diff
@@ -431,8 +431,9 @@ module IO {
       (ie, they can open up channels that are not shared).
 */
 
-private use SysBasic, SysCTypes;
-use SysError;
+use SysBasic, SysCTypes;
+public use SysError;
+public use SysBasic;
```

Comment:
A public and a private use of `SysBasic`?

Reply:
Thanks for pointing that out.